.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.m
```

Comment:
- [ ] Kindly ensure that your `test.md` file is correctly ignored, containes the `"This file should be ignored by git"` text and it doesn't appear on your github repository. I can see that you ignored a file different to expected one as you wrote `test.m` instead of `test.md`. Please make sure you fix that for **the project's requirements**